remediation_worker/jobs/azure_blob_remove_public_access/azure_blob_remove_public_access.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2020 VMware Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import sys
+import logging
+
+from azure.mgmt.storage import StorageManagementClient
+from azure.common.credentials import ServicePrincipalCredentials
+from azure.mgmt.storage.models import PublicAccess
+
+logging.basicConfig(level=logging.INFO)
+
+
+def logcall(f, *args, **kwargs):
```

Comment:
These `logcall` functions are never used. We should remove them if this is in fact the case.

Reply:
We will take that removal in the next PR